perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -264,7 +274,12 @@ sub dashboard_export_problems_add_columns {
     $csv->csv_extra_data(sub {
         my $report = shift;
 
-        return {
+        my $extra = $report->get_extra_metadata;
+        my %bulky_items = ('item_1' => '', 'item_2' => '', 'item_3' => '', 'item_4' => '', 'item_5' => '', 'item_6' => '',
+        'item_7' => '', 'item_8' => '', 'item_9' => '');
+        %bulky_items = (%bulky_items, map {$_ => $extra->{$_}} grep { $_ =~ /^(item_\d+)$/ } keys %$extra);
```

Comment:
Similarly I wonder if this might be better as a loop `for (1..11) { $bulky_items{"item_$_"} = $extra->{$_} || "" }` type thing?